src/main/frontend/fs/sync.cljs

```diff
@@ -3469,6 +3489,14 @@
              (when (nil? n)
                (<sync-stop))))
 
+;; try to re-start sync when state=stopped every 1min
+(go-loop []
+  (<! (timeout 60000))
+  (when (and @network-online-cursor       ; is online
+             (= ::stop (:state (state/get-file-sync-state))))
+    (println "trying to restart sync...")
+    (<sync-start))
+  (recur))
```

Comment:
Add a go-loop to try to restart sync when it's possible (network status is online & sync-state is ::stop)